plugin/src/main/java/net/dzikoysk/funnyguilds/listener/EntityDamage.java

```diff
@@ -49,20 +50,23 @@ public void onDamage(EntityDamageByEntityEvent event) {
                     return;
                 }
 
-                if (victimUser.getGuild().equals(attackerUser.getGuild())) {
-                    if (!victimUser.getGuild().getPvP()) {
+                Guild victimGuild = victimUser.getGuild().get();
+                Guild attackerGuild = attackerUser.getGuild().get();
+
+                if (victimGuild.equals(attackerGuild)) {
```

Comment:
Tak samo jak wyżej

Reply:
To sie nie sypnie bo wyżej jest check czy oboje mają gildie